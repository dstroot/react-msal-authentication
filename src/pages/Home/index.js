import React from "react";

const Home = () => (
  <div className="container mt-5">
    <div className="row">
      <div className="col">
        <h2>Home Page</h2>
        <h1 className="display-4">Hi there!</h1>
      </div>
    </div>
  </div>
);

export default Home;
