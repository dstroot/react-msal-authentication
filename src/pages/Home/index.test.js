import React from "react";
import renderer from "react-test-renderer";

// import component to test
import Home from "../Home";

describe("Home", () => {
  it("it should render", () => {
    const component = renderer.create(<Home />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
