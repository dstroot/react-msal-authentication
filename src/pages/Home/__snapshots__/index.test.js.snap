// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home it should render 1`] = `
<div
  className="container mt-5"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <h2>
        Home Page
      </h2>
      <h1
        className="display-4"
      >
        Hi there!
      </h1>
    </div>
  </div>
</div>
`;
